src/calculator/index.ts

```diff
@@ -186,12 +186,22 @@ export default class Calculator {
       round.metadata?.quadraticFundingConfig?.sybilDefense ??
       false;
 
+    // 1. convert threshold amount to 6 decimals
+    // 2. truncate rest of decimals to bigint
+    // 3. convert decimals to token decimals
+    // 4. remove initial 6 decimals
     const minimumAmount =
```

Comment:
This works as expected - but for readability we could choose to put the nested operations in its own function. Something like: 
```